sql/src/main/java/io/crate/operation/fetch/FetchContext.java

```diff
@@ -97,6 +112,16 @@ public void innerPrepare() {
         }
     }
 
+    @Override
+    protected void innerStart() {
+        int c = 0;
+        for (Collection<Reference> fetchRef : phase.fetchRefs()) {
+            c += fetchRef.size();
+        }
+        if (c == 0) {
```

Comment:
consider adding a comment here explaining why this case is valid, because on a first glance it looks like a planner error